models/product/src/product-data/category-order-hint/fields-config.ts

```diff
@@ -0,0 +1,11 @@
+import { fake, type TModelFieldsConfig } from '@commercetools-test-data/core';
+import type { TCategoryOrderHintGraphql } from './types';
+
+export const graphqlFieldsConfig: TModelFieldsConfig<TCategoryOrderHintGraphql> =
+  {
+    fields: {
+      __typename: 'CategoryOrderHint',
+      categoryId: fake((f) => f.string.uuid()),
+      orderHint: fake(() => String(Math.random())),
```

Comment:
As far as I saw, `orderHint` should be between 0 and 1 and what it has been using is this `fake((f) => f.number.float({ min: 0.01, max: 0.99 }).toString())`

Reply:
`Math.random()` returns a value between 0 and 1